LondonBritishLibrary/orient/BLorient13264.xml

```diff
@@ -0,0 +1,327 @@
+                                <item xml:id="a4">
+                                    <locus target="#133va"/>
+                                    <title ref="LIT3058RepCh338"/>
+                                    <desc type="GuestText"/>
+                                    <q xml:lang="gez">እሰግድ፡ ለኪ፡ ወእዌድስኪ፡ ኦእግዝእትየ፡ ማርያም፡ በከመ፡ ወደስኪ፡ በድርሳኑ፡ ዮሐንስ፡ አፈ፡ ወርቅ፡ እንዘ፡ ይብል፡ ተፈስሒ፡ 
+                                        ኦእግዝእትየ፡ <supplied reason="omitted" resp="Strelcyn">ማርያም፡</supplied></q>
+                                </item>
+                                <item xml:id="e1">
+                                    <locus target="#1 #1v #2r #3r #142r #149vb"/>
+                                    <desc type="Unclear">Short notes, pen trials.</desc>
+                                </item>
+                                <item xml:id="e2">
+                                    <locus target="#3r #6r #11v #12v #14r #14v #15r #126r"/>
+                                    <desc type="Unclear">In the margins there are many notes written in a very small and fine script by at least three
+                                        different hands, some in Amharic.</desc>
+                                    <q xml:lang="am"/>
+                                </item>
+                                <item xml:id="e3">
+                                    <desc type="OwnershipNote">After <locus target="#124"/> there has been sewn a narrow piece of vellum containing
+                                        the name of the owner, <persName role="owner" ref="PRS14335GabraHeywat"><roleName type="title">dabtara
+                                        </roleName>Gabra Ḥǝywat</persName> and his son 
+                                        <persName ref="PRS14336Yohannes">Yoḥannǝs</persName>.</desc>
+                                </item>
+                            </list>
+                        </additions>
+                        <bindingDesc>
+                            <binding xml:id="binding">
+                                <decoNote xml:id="b1"/>
+                                <decoNote type="bindingMaterial" xml:id="b2"><material key="wood"/></decoNote>
+                                <decoNote xml:id="b3" type="Cover"><material key="leather">Stamped leather.</material></decoNote>
+                                <decoNote xml:id="b4" type="SlipCase">Contained in a double hide case.</decoNote>
+                            </binding>
+                        </bindingDesc>
+                    </physDesc>
+                    <history>
+                        <origin>
+                            <origDate notBefore="1600" notAfter="1799">17th to 18th century</origDate>
+                        </origin>
+                        <provenance>According to <bibl><ptr target="bm:Strelcyn1978BritishLibrary"/>
+                            <citedRange unit="page">9-10</citedRange></bibl> taken from King Theodore's Library in Magdala in 
```

Comment:
mark down  `<persName ref="PRS9430Tewodros"/>` and `<placeName ref="INS0101MadhaneAlam"/>`